### ISSUE TITLE
Folders

### ISSUE THREAD

**Issue by Fleker** [ITVlab/TvAppRepo#38]
What if I could generate shortcuts with a specific Intent to an activity in this app. This Intent would load a specific list of apps, much like a folder.

**Comment by Xstar97** [comment]
like a category?

**Comment by Fleker** [comment]
I'm thinking of something akin to folders on a phone, where it could contain a collection of apps. If an intent could embed a unique identifier for each shortcut, then when the user opens that folder shortcut, it would open an activity that loads the apps associated with that identifier

**Comment by Xstar97** [comment]
so basically like any launcher on android does?
So this folder will added to the homescreen & when opened it would launch an intent to load a specific list of apps in it?

sorry just trying to get a better understanding of your features.

**Comment by Fleker** [comment]
Yes. You would setup a folder in the app. Something like "Music" and add
apps. This is my first folder (index 0). This would generate an intent to
my Folder activity with the additional field of index.

When the folder activity opens, as a side panel or pop-up, it would take
that index, find all the activities, and let you pick one to open.

That's the idea. I'm not sure if it would be a good idea.

**Comment by Xstar97** [comment]
I think its a good idea because it can add a mini "category" or folder in a larger category which in turn will free up more space so users will spend less time searching for their apps. This feature will be userful to even gamers who install a ton of games(i use eulators more often..so I would love to separate the games from my emulators.)
keep up good work & I hope to see this feature implemented soon!

suggestion: you might want to implement a side panel & pop up for user specfic customizations..I think either one will work though.
